tensorflow/lite/kernels/depthwise_conv.cc

```diff
@@ -398,6 +406,35 @@ TfLiteStatus EvalQuantizedPerChannel(TfLiteContext* context, TfLiteNode* node,
   return kTfLiteOk;
 }
 
+TfLiteStatus EvalQuantizedPerChannel16x8(
+    TfLiteContext* context, TfLiteNode* node, TfLiteDepthwiseConvParams* params,
```

Comment:
nit: please remove unused arguments like node or context, and make arguments const wherever possible.

Reply:
done